cypress/CYPRESS-README.md

```diff
@@ -1,7 +1,7 @@
 
 # Best Practices
 
-In order to write a realiable cypress test suites, there are some best practices we should follow that our outlined in the cypress documentation and also some best practices we have learned from trying to write realiable tests.
+In order to write a reliable cypress test suites, there are some best practices we should follow that our outlined in the cypress documentation and also some best practices we have learned from trying to write reliable tests.
```

Comment:
![Thank you](https://i0.wp.com/allthecubs.com/wp-content/uploads/2022/01/thank-you-gif-the-office-steve-carell.gif)